### PR TITLE
Rename unifying plugin to logitech-hid

### PR DESCRIPTION
**Pull request by superm1**
Type of pull request:
- [ ] New plugin (Please include [new plugin checklist](https://github.com/hughsie/fwupd/wiki/New-plugin-checklist))
- [x] Code fix
- [ ] Feature
- [ ] Documentation

### CODE REVIEW

File: plugins/logitech-hid/fu-logitech-hid-hidpp-msg.c [hunk 1]
```diff
@@ -8,17 +8,17 @@
...
 
-#include "fu-unifying-hidpp.h"
-#include "fu-unifying-hidpp-msg.h"
+#include "fu-logitech-hid-hidpp.h"
+#include "fu-logitech-hid-hidpp-msg.h"
 
-FuUnifyingHidppMsg *
-fu_unifying_hidpp_msg_new (void)
+FuLogitechHidHidppMsg *
+fu_logitech_hid_hidpp_msg_new (void)
```
Comment (hughsie) [inline plugins/logitech-hid/fu-logitech-hid-hidpp-msg.c]: hid_hidpp looks kinda odd to me -- maybe `FuLogitechHidppMsg` instead?

Comment (hughsie) [reply plugins/logitech-hid/fu-logitech-hid-hidpp-msg.c]: Aaaaactually, I wonder if the plugin name should be logitech-hidpp?

Comment (FFY00) [reply plugins/logitech-hid/fu-logitech-hid-hidpp-msg.c]: It will also have the bootloader protocols, but I guess that's pretty small.

Comment (superm1) [reply plugins/logitech-hid/fu-logitech-hid-hidpp-msg.c]: I just pushed a fixup to rename to logitech-hidpp instead then.

**Comment by FFY00** [comment]
Will this require any change in the remote metadata?

**Comment by superm1** [comment]
> Will this require any change in the remote metadata?

I don't think so.  It still identifies the protocol as `com.logitech.unifying` and `com.logitech.unifyingsigned` which was the only interaction I expected between the two.

**Pull request merged**